TicketBooking/Flight/Services/FlightPrinter.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TicketBooking.Class;
+using TicketBooking.Flight.FlightModel;
+
+namespace TicketBooking.Flight.Services;
+
+public class FlightPrinter
+{
+    public static void DisplayFlights(List<FlightModel.Flight> flights)
+    {
+        Console.WriteLine($"Available flights: ");
+
+        foreach (var flight in flights)
+        {
+            Console.WriteLine(flight.ToString());
+        }
+    }
+
+    public static string GetFlightDetails(FlightModel.Flight flight)
+    {
+        StringBuilder stringBuilder = new StringBuilder();
+        stringBuilder.AppendLine(flight.ToString());
+        stringBuilder.AppendLine(GetFlightClassesAndPrices(flight));
+        return stringBuilder.ToString();
```

Comment:
No need to use a StringBuilder here. Since we are not iterating over the data or making validations, we don't come across performance issues of instantiating strings every time. Just return the interpolated string directly.